src/main/java/org/frc571/bradley/commands/IntakeCommand.java

```diff
@@ -24,28 +24,35 @@ public IntakeCommand() {
     // Called when the command is initially scheduled.
     @Override
     public void initialize() {
-        intake.setSpeed(SpeedConstants.kIntakeMotorSpeed);
+        LowerIntake lower = new LowerIntake();
+        while (!lower.isFinished()) {
```

Comment:
This will lock the entire robot up while intake is lowering. No other code will be executed. This presents a problem.

If `lower.isFinished()` never returns `true`, the program will halt forever. Please find a different way to do this.

There is code that is called by the scheduler that you can use to check the status of the lower intake without blocking the execution of the code. Perhaps one of those classes is a better place to do it?

Either way, blocking like this should be avoided.